### ISSUE TITLE
Ops! Something went really wrong and regionfixer crashed Exception:<class 'AssertionError'>Unrecognized scanned chunk in scan_chunk().

### ISSUE THREAD

**Issue by BullShark** [Fenixin/Minecraft-Region-Fixer#171]
**Describe the bug**
Command I ran when the script crashed: python3 regionfixer.py --fc --fm --fw -p 8 -l summary_fix_report.log /home/bullshark/CraftBukkit/world/

**Full copied text from the MS-DOS view**
Please, include the command used to run regionfixer.

```
[bullshark:~/Minecraft-Region-Fixer]$ python3 regionfixer.py --fc --fm --fw -p 8 -l summary_fix_report.log /home/bullshark/CraftBukkit/world/                                                                                     (master)

Welcome to Region Fixer!
(v 0.3.4)

############################################################
#################  Scanning world: world  ##################
############################################################

World info:
- 12 region/level files,
- 3 POI files,
- 6 entities files,
- 7 player files,
- and 1 data files.

-------------------- Checking level.dat --------------------
'level.dat' is readable

---------------- Scanning UUID player files ----------------
7 of 7|#######################################################################################################################################################################################################################|Time: 0:00:00

------------- Scanning old format player files -------------
Info: No files to scan.

---------- Scanning structures and map data files ----------
1 of 1|#######################################################################################################################################################################################################################|Time: 0:00:00

--------- Scanning region, POI and entities files ----------
0 of 21|                                                                                                                                                                                                                     |ETA:  --:--:--

Ops! Something went really wrong and regionfixer crashed.


Bug report:

**********
*** Exception while scanning:
*** r.-1.-1.mca
**********
*** Printing the child's traceback:
*** Exception:<class 'AssertionError'>Unrecognized scanned chunk in scan_chunk().
**********
*** File /home/bullshark/Minecraft-Region-Fixer/regionfixer_core/scan.py, line 813, in scan_region_file
***   entity_limit)**********
*** File /home/bullshark/Minecraft-Region-Fixer/regionfixer_core/scan.py, line 996, in scan_chunk
***   raise AssertionError("Unrecognized scanned chunk in scan_chunk().")
**********

[bullshark:~/Minecraft-Region-Fixer]$
```

**Expected behavior**
To get 95% of what I had in my world back

**Screenshots**
![image](https://user-images.githubusercontent.com/1526595/144717653-bb35b533-c895-4b55-88c8-49e4a41bdb23.png)

**Files that would help solving the issue**
If possible, the world/files that triggers the error.
https://expectusafterlun.ch/world.tbz

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Manjaro with KDE latest
 - Python version: [e.g. 2.7] Python 3.7.3
 - Region Fixer Version [e.g. 2.0.1] latest git clone

**Additional context**

**Comment by 734F96** [comment]
Hello,
Thank you for this very clean issue.
This is because you have a 1.18 world. The support of this version is currently in PR #169.
Have a nice day ! ^^

**Comment by Fenixin** [comment]
Hello!

I have just merged #169 from @734F96. I'm closing this, feel free to open/reopen if you have any problem.
